src/main/groovy/com/getkeepsafe/dexcount/DexMethodCountTask.groovy

```diff
@@ -350,7 +348,6 @@ class ModernMethodCountTask extends DexMethodCountTaskBase {
     }
 }
 
-@ParallelizableTask
```

Comment:
This was removed in Gradle 4.0 - https://discuss.gradle.org/t/parallelizabletask-is-removed-in-gradle-4-0/22970/10.